lib/github_client.rb

```diff
@@ -0,0 +1,30 @@
+class GithubClient
+  def initialize(user, client = Octokit::Client)
+    @user = user
+    @client = client
+  end
+
+  def populate_user_repos
+    response = get_user_repos
+    response.each do |repo|
+      Repo.
+        find_or_create_by!(
+          user: user,
+          name: repo["name"],
+          full_name: repo["full_name"],
+        )
+    end
+  end
+
+  private
+
+  attr_accessor :user
```

Comment:
I would make this an `attr_reader` unless you plan on allowing on allowing something in this class to change `@user` later.